routes/login/confirm-device.js

```diff
@@ -129,30 +113,42 @@ exports.resendCode = async (req, res, next) => {
 exports.removeDevice = async (req, res) => {
   const { id } = req.body;
   const { referer } = req.headers || {};
-  const { uuid, language } = req.session;
+  const { uuid, language, is_trusted } = req.session;
 
-  const referer_url = new URL(referer || `/${language}/edit/contributor?id=${uuid}`)
-	const referer_params = new URLSearchParams(referer_url.search)
+  const referer_url = new URL(
+    referer || `/${language}/edit/contributor?id=${uuid}`
```

Comment:
`/` is not enough to create a url.
<img width="449" alt="Screenshot 2023-10-10 at 9 44 08 AM" src="https://github.com/UNDP-Accelerator-Labs/platform/assets/533142/ce50f546-0089-4be9-a818-f2e51ce11477">